### DISCUSSION TITLE
Some ideas to improve the user experience with the Elegoo Neptune 3

### DISCUSSION THREAD

**Discussion by Ok2dot0** [NARUTOfzr/Neptune_3#52] [Ideas]
### I have four ideas to improve the user experience.

The first is a thumbnail of the part when you are choosing a model from the SD-card. It would be also nice if it will be displayed when the part is printing.

And the second is a better readability of the filename while printing and also choosing a model from the card. For example: while printing the name is scrolling from left to right and while choosing the name is shorten as now. But when you choose it and the screen to confirm pop up, the name expands to full size and the thumbnail is shown.

Third, please fix the "print" string at the top of the screen while printing. Maybe change it to "Printing..." or some similar.

Fourth, please add a estimating time info somewhere while printing.

I, and for sure other users, will be very happy. And thank you for caring about the most important thing of a 3D-printer.

Many Thanks.

**Comment by ghost** [comment]
some other things to add is lin adv acceleration jerk max speed per axis and printing speed
